drivers/net/wireless/marvell/mwifiex/main.h

```diff
@@ -1054,6 +1054,9 @@ struct mwifiex_adapter {
 	void *devdump_data;
 	int devdump_len;
 	struct timer_list devdump_timer;
+
+	/* Indicates a reset is ongoing or not */
+	bool pci_reset_ongoing;
```

Comment:
Urgh, I didn't think deeply when I added this.
I added stuff that will be used only by PCIe into `mwifiex_adapter` (common to all the interfaces).
So, better to add into `pcie_service_card` instead.

Reply:
Oh right, that makes sense.